issue-1

alorenz-raptor opened the pull request:
Fixes #1

Commit:
AL25Nov25_1

2 digits done. absolutely gorgeous scan. using bounding box -> negative lasso -> pos+neg point and things are coming out clean.

Commit:
AL25Nov25_2

digits coming along nicely.

Commit:
AL25Nov25_3

metacarps and phalanges of right manus complete- maybe missing an ungual? very cartilaginous and unclear.

Commit:
AL26Nov25_1

following Pashchenko 2022 Fig 1 for numbering of digits to assist work on the carpals. added some segments for the epiphyses.

Commit:
AL26Nov25_2

completely forgot the fifth digit- did that and checked the more distal and unclear phalanges... need to check number on digit IV. continuing to make an ROI for the forelimb so i can do some easy bones as these cartilaginous bits are tricky.

Commit:
AL26Nov25_3

zeugopod yay. having some issues where the shift is not letting me align slice views, pushing so i can close and reopen slicer.

Commit:
AL26Nov25_4

going to work on the humerus and shoulder girdle next.

Commit:
AL1Dec25_1

humerus and the shoulder girdle of the right limb. spent a little time finding and looking over some limb anatomy, found good sources for the distal forelimb (zeugopod+) but not much for the shoulder girdle. im going to do the left manus. left a lot of cartilaginous bits unsegmented on this side. on left, will segment bone and cartilage caps together in the same segment and see which looks better.

Commit:
AL1Dec25_2

segmentation is starting to throw error messages. getting through the metacarpals and phalanges while trying to include the cartilaginous caps. push to troubleshoot the NNi failures.

Commit:
AL1Dec25_3

same NNi failure which started the same way: i misclicked with positive point prompt on in the 3D view while holding and dragging. recreated so noting the same failure trigger. closing the terminal and relaunching NNi server worked the first time, but i had also closed slicer. looks like it was right to do this as relaunching without closing makes the markup glitch occur where the prompts are only treated as markup items.

Commit:
AL1Dec25_4

renaming a lot of these bones so they are identifiable later. taking a break. this session i found that repeatable error that might not be in the newer versions of Slicer. im breaking and then coming back on the other instance to try and stress test the new instance and more recent slicer ver with this same error.

Commit:
AL2Dec25_1

humerus, radius, ulna of left forelimb. next the shoulder girdle and some renaming.

Edits:

Added segments:
- Segment_8
- humerus L
- ulna L
Removed segments:
- to redo

Commit:
AL2Dec25_2

left shoulder girdle, more renaming and rearrangement. next, get the sternum and check i didn't miss epiphyses on the R manus. very close to done with this one.

Edits:

Added segments:
- coracoid L
- scapula L

Commit:
AL2Dec25_3

cartilage between coracoids and scapulae finished. looking for sternum.

Edits:
Modified segments:
- coracoid R
- scapula R
Added segments:
- shoulder cartilage L
- shoulder cartilage R

Commit:
AL2Dec25_4

pulled out a very faint sternum element, nearly indistinguishable from the tissues around it. going to polish up epiphyses on the right manus then submit a PR review.

Edits:

Added segments:
- sternum?

Commit:
AL2Dec25_5

epiphyses is just way too much to go for separately of the ossified portion. left the ones i had done already as segments, but going to call this my finished draft for now. push to save then PR review.

Edits:

Added segments:
- metacarpal i dist epiphysis R
- phalanx iv.iii L
- phalanx iv.v L
- sternum
Removed segments:
- phalanx iv.iv L

dinonoto reviewed (changes requested):
Fix ridge on distal right humerus

Commit:
AL5Jun26_1

distal end of right humerus cleaned as requested, ready for producing simplified segmentation.

Edits:
Modified segments:
- humerus R

Commit:
AL8Jun26_1

Used NNi to get the missing metacarpal ii prox epiphysis. pushing to save the segment, then one more push incoming for the reorganization of segments before PR review.

Edits:

Added segments:
- metacarpal ii prox epiphysis
Removed segments:
- metacarpal i dist epiphysis R

Commit:
AL8Jun26_2

renamed and sorted

Edits:

Added segments:
- metacarpal ii prox epiphysis R
Removed segments:
- metacarpal i dist epiphysis R
